packages/components/src/vue/components/lookup.vue

```diff
@@ -17,17 +17,13 @@
         </span>
 
         <div class="alpheios-lookup__search-control">
-          <input
-              autocapitalize="off"
-              autocorrect="off"
-              @keyup.enter="lookup"
-              @keyup="updateBetaCodes"
-              class="alpheios-input"
-             :class="{ 'alpheios-rtl': directionRtl}"
-              type="text"
-              v-model="lookuptext"
-              id="alpheios-lookup-form-input"
-          >
+          <input-autocomplete
```

Comment:
I would like to make this optional, based upon a configuration setting (sorry!)

I'd like a setting in alpheios-feature-options:

```
"enableLogeionAutoComplete": {
      "defaultValue": true,
      "labelText": "Enable Autocomplete with <a href="https://logeion.uchicago.edu" target="_blank">Logeion</a>",
      "boolean": true,
      "values": [
        {
          "value": true,
          "text": "Yes"
        },
        {
          "value": false,
          "text": "No"
        }
      ]
    }
```

And I'd like it to be presented as a checkbox option below the lookup box when the selected language is Latin or Greek